CMakeLists.txt.user.18

```diff
@@ -1,1627 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE QtCreatorProject>
-<!-- Written by QtCreator 4.2.0, 2017-01-18T10:53:40. -->
```

Comment:
@pold500  Please squash your commits and group your changes (for example you can handle one class per commit: "class.cc" "class.h" "files_related_to_this_class"), And give more complex and deep description in your commit messages: describe what has been done in each commit. If you make temporary commit please write something like this: "Changes related to commit named 'one_that_is_already_existing' ", all temporary commits should be squashed with related commits at the end of review. Currently it is hard to navigate through your changes.